src/views/Landing.js

```diff
@@ -143,7 +219,9 @@ function Landing() {
 function FeatureCard({ feature }) {
   return (
     <Card
-      onClick={() => (window.location = 'https://google.com')}
+      onClick={() =>
+        window.open('https://google.com', '_blank', 'noopener noreferrer')
+      }
```

Comment:
go ahead and make this a history.push thing (can just pick a random resource for now)